audit/values.schema.json

```diff
@@ -2,33 +2,6 @@
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "additionalProperties": false,
     "properties": {
-        "authorizationService": {
-            "properties": {
-                "dnsName": {
-                    "type": "string"
-                }
-            },
-            "type": "object"
-        },
-        "cacheService": {
-            "properties": {
-                "dnsName": {
-                    "type": "string"
-                }
-            },
-            "type": "object"
-        },
-        "configurationService": {
-            "properties": {
-                "dnsName": {
-                    "type": "string"
-                },
-                "port": {
-                    "type": "integer"
-                }
-            },
-            "type": "object"
-        },
         "global": {
```

Comment:
can we move some of the global stuff up to the datawave-monolith-umbrella level?

Reply:
probably yes -- eventually. i want to hold off on too many more changes in this pr, but happy to address more in another.